com.unity.netcode.gameobjects/Tests/Runtime/NetworkTransform/NetworkTransformStateTests.cs

```diff
@@ -269,6 +274,7 @@ public void TestSyncAxes([Values] SynchronizationType synchronizationType, [Valu
 
                 if (syncPosX || syncPosY || syncPosZ || syncRotX || syncRotY || syncRotZ || syncScaX || syncScaY || syncScaZ)
                 {
+                    Assert.IsTrue(networkTransform.NetworkManager != null, "NetworkManager is NULL!");
```

Comment:
You could use `Assert.NotNull(networkTransform.NetworkManager)`
or `Assert.That(networkTransform.NetworkManager, Is.Not.Null)`

Reply:
Ahh... true that!  I would think that would help clarify the context faster for reviewers.
(nice tip)
👍